packages/create-redwood-app/src/create-redwood-app.js

```diff
@@ -98,36 +74,24 @@ if (!targetDir) {
 
 const newAppDir = path.resolve(process.cwd(), targetDir)
 const appDirExists = fs.existsSync(newAppDir)
+const templateDir = path.resolve(__dirname, '../template')
```

Comment:
I'm a bit concerned about this creating an implicit dependency with the file-structure. If the path inside `dist` is not the expected one, `templateDir` will point to the wrong directory. Any ideas around how to improve this?

Reply:
This doesn't concern me much, but maybe something like https://www.npmjs.com/package/app-root-path is helpful.

Reply:
I sound like Shania Twain 🤠

Reply:
The amount of packages in NPM never ceases to surprise me. I have to look up a package for preparing a coffee 😆